packages/components/src/utils/hooks/use-cx.ts

```diff
@@ -20,6 +20,8 @@ const EmotionCacheContext: Context< EmotionCache > = CacheProvider._context;
 const useEmotionCacheContext = () => useContext( EmotionCacheContext );
 
 const isSerializedStyles = ( o: any ): o is SerializedStyles =>
+	// eslint-disable-next-line eqeqeq
+	o != null &&
```

Comment:
`ClassNamesArgs` includes `null` and `undefined` so we need to be careful of those here lest we cause a NPE